src/App.tsx

```diff
@@ -110,14 +120,41 @@ function HomeScreen() {
     });
   };
 
+  const submit = () => {
+    const items = Object.keys(state.items).map(itemName => {
+      const itemDetails = state.items[itemName];
+      return {
+        name: itemName,
+        cost: itemDetails.price,
+        people: itemDetails.purchaserList
+      }
+    });
+
+    try {
+      submitTransaction({
+        items, 
+        tip: state.tip,
+        tax: state.tax,
+        party: state.party
+      });
+
+      dispatch({ type: DispatchActions.RESET });
+      if (taxInputRef.current) {
+        taxInputRef.current.clear();
+      };
+    } catch {
+      // TODO: show error message
```

Comment:
oo yea we should create a ticket to figure out how to surface this error message